Assets/MixedRealityToolkit/Utilities/Gltf/Schema/GltfMeshPrimitive.cs

```diff
@@ -53,7 +53,7 @@ public class GltfMeshPrimitive : GltfProperty
         /// A dictionary object, where each key corresponds to mesh attribute semantic
         /// and each value is the index of the accessor containing attribute's data.
         /// </summary>
-        public GltfMeshPrimitiveAttributes Attributes { get; internal set; }
+        public Dictionary<string, int> Attributes { get; internal set; }
```

Comment:
Same comment as https://github.com/microsoft/MixedRealityToolkit-Unity/pull/7490#discussion_r388562897.

Reply:
This one feels like it might be easier to deprecate the existing property and create a new one than the other comment, though we wouldn't be able to reuse the name.

Reply:
What would the deprecated property do though? Wouldn’t it be fully broken? Also, what would we name the new property?

Reply:
What would the deprecated property do though? Wouldn’t it be fully broken? Also, what would we name the new property?

Reply:
Hmm...possibly, if we keep filling both properties in the short term, it'd allow people to migrate if needed? Basically, keep https://github.com/microsoft/MixedRealityToolkit-Unity/pull/7490/files#diff-05951bac84393068d2888d4927ca4b57L202 around, but deprecate this property?
@davidkline-ms, what do you think? Might be more trouble than it's worth perf-wise to deserialize the glTF basically twice to keep this property around.

Reply:
We could change this to an attribute, and return the ‘ GltfMeshPrimitiveAttributes’ instance hydrated with data from the dictionary.

Reply:
@keveleigh, is it possible to maintain both without deserializing twice by copying the objects?

Reply:
@davidkline-ms yeah, it looks like @FreakTheMighty had a good idea on that above